core-primitives/test/src/mock/onchain_mock.rs

```diff
@@ -45,6 +53,11 @@ impl OnchainMock {
 		self.with_storage_entries(into_key_value_storage(set))
 	}
 
+	pub fn with_mr_enclave(mut self, mr_enclave: [u8; SGX_HASH_SIZE]) -> Self {
```

Comment:
Following my usual argument of having words separate, I would prefer to have `mr_enclave`, since `mr` is already an abbreviation for `measurement result`.